Cargo.toml

```diff
@@ -11,18 +11,18 @@ keywords = ["wayland", "client"]
 description = "Toolkit for making client wayland applications."
 
 [dependencies]
-bitflags = "1.0"
+bitflags = "1.2"
```

Comment:
There is no need to update minor versions of other dependencies like that, cargo already considers these versions as compatible with each other, so `cargo update` will already pull them.

Reply:
I've reverted these back and only updated `andrew`.